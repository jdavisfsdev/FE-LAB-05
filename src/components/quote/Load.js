import React from 'react';
import style from './Load.module.css';

export default function({ onClick }) {
  <button onClick={onClick}>LOAD</button>;
}
