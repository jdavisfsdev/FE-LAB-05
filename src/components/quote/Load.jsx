import React from 'react';
import style from './Load.module.css';

export function Load({ onClick }) {
  return (
    <button data-testid="button" onClick={onClick}>LOAD</button>
  );
}
