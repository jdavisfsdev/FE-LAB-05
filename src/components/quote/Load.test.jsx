import React from 'react';
import { render } from '@testing-library/react';
import { Load } from './Load';

describe('Load component snappy', () => {
  it('renders LOAD button', () => {
    const { asFragment } = render(
      <Load />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
