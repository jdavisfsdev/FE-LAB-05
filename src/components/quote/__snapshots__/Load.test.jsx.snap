// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load component snappy renders LOAD button 1`] = `
<DocumentFragment>
  <button
    data-testid="button"
  >
    LOAD
  </button>
</DocumentFragment>
`;
